ProjectLighthouse/Controllers/UserController.cs

```diff
@@ -39,7 +40,7 @@ public async Task<IActionResult> GetUser(string username)
             if (token == null) return this.StatusCode(403, "");
 
             string? user = await this.GetSerializedUser(username, token.GameVersion);
-            if (user == null) return this.NotFound();
+            if (user == "") return this.NotFound();
```

Comment:
`if(string.isNullOrEmpty(user))`

Reply:
TIL, that's handy. Thanks. https://github.com/LBPUnion/project-lighthouse/pull/62/commits/d36a7d576560517cecfc6260661c681a5caf82be